int32 waypoint_number
---
bool result